models/user.js

```diff
@@ -1,7 +1,9 @@
 'use strict'
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
-  }, {
+    account: {
+      type: DataTypes.STRING
+    },
```

Comment:
I think we also need to add attribute "cover" for user cover photo. npm run test will show error of missing attribute.